src/docs/tutorials/DataPlantOnboarding.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: docs
```

Comment:
I like the `author_orcid` and `author_github` 👍

Reply:
They made it to the yaml in README last week already :) 

https://github.com/nfdi4plants/nfdi4plants.knowledgebase/blob/main/src/docs/README.md